Cargo.toml

```
@@ -57,3 +58,8 @@ members = [
     "tikv-client-store",
     "mock-tikv"
 ]
+
+[[test]]
+name = "failpoint_tests"
+path = "tests/failpoints.rs"
```

```suggestion
path = "tests/failpoints_tests.rs"
```
And I prefer `failpoint_tests` :)